src/components/Player.js

```diff
@@ -24,38 +20,31 @@ const Player = ({
     animationPercentage: 0,
   });
   const [songVolume, setSongVolume] = useState(50);
-
-  const activeLibraryHandler = (nextPrev) => {
-    const newSongs = songs.map((song) => {
-      if (song.id === nextPrev.id) {
-        return {
-          ...song,
-          active: true,
-        };
-      } else {
-        return {
-          ...song,
-          active: false,
-        };
-      }
-    });
-    setSongs(newSongs);
-  };
```

Comment:
Ta funkcja w gruncie rzeczy jest niepotrzebna. Rozumiem, że w stan `songs` w `App.js` został dodany tylko po to, żeby mów oznaczać jedną z piosenek w tej tablicy jako aktywną. Tylko czy warto zarządzać kolejną kopią danych, gdzie tylko jeden atrybut jednego z elementów się zmienia? Spokojnie można to zastąpić rozsądniejszym wykorzystanie `currentSong`.